README.md

```diff
@@ -931,6 +932,7 @@ so reviewdog will use [Check annotation](https://docs.github.com/en/rest/checks/
 | **`gitlab-mr-commit`**       | OK      | Partially Supported [2] | Partially Supported [2] | Partially Supported [2] |
 | **`gerrit-change-review`**   | OK      | OK? [3]        | OK? [3]                 | Partially Supported? [2][3] |
 | **`bitbucket-code-report`**  | NO [4]  | NO [4]         | NO [4]                  | OK |
+| **`gitea-pr-review`**        | OK      | OK? [3]        | OK? [3]                 | OK |
```

Comment:
It should be OK, OK, Partially Supported [2], Partially Supported [2]

Reply:
@haya14busa fixed